### PR TITLE
Added README file content

### PR DESCRIPTION
**Pull request by dashaluna**
Issue #33

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,36 @@
-# hm-content-toc
-WordPress plugin. Creates content TOC for specified header elements from the current post content. Adds anchor elements in the content just before the matched headers.
+[![Build Status](https://travis-ci.org/dashaluna/hm-content-toc.svg?branch=master)](https://travis-ci.org/dashaluna/hm-content-toc)
+
+=== Plugin Name ===
+Contributors: dashaluna, tcrsavage, johnbillion, sanchothefat
```
Comment (dashaluna) [inline README.md]: @tcrsavage @johnbillion @sanchothefat let me know if you want me to delete your username from here :) Not sure what this affects..

Comment (roborourke) [inline README.md]: Nice of you to mention us but not necessary, happy for you to leave me in though :)

File: README.md [hunk 2]
```diff
@@ -1,2 +1,80 @@
...
+
+== Changelog ==
+
+== Upgrade Notice ==
+
+== Translations ==
+
+* English - default, always included
+
+*Note:* All my plugins are localized/translateable by default. This is very important for
```
Comment (johnbillion) [inline README.md]: Should be "translatable"

File: README.md [hunk 3]
```diff
@@ -1,2 +1,80 @@
...
+all users worldwide. So please contribute your language to the plugin to make it even more useful.
+For translating I recommend ["Codestyling Localization" plugin](http://wordpress.org/extend/plugins/codestyling-localization/)
```
Comment (johnbillion) [inline README.md]: The "Codestyling Localization" plugin doesn't exist.

**Comment by johnbillion** [comment]
As discussed, this file needs headings with `##` notation instead of `==` notation.

**Comment by dashaluna** [comment]
@johnbillion done, will you show me how to generate the `readme.txt` before merging this?

**Pull request merged**